web/src/app/shared/components/type-ahead/type-ahead.component.html

```diff
@@ -1,48 +1,19 @@
-<div *ngIf="dataSource" class="custom-form-control" [ngStyle]="{'width': width ? width : auto }">
-  <ng-template class="wrapper-text" #customItemTemplate let-model="item" let-index="indexTemplate" let-query="query">
-    <div (mousedown)="typeaheadOnSelect({'item':model})">
-      <div>
-        <span [innerHTML]="join(model.accessId, query)">
-        </span>
-      </div>
-      <div>
-        <span [innerHTML]="join(model.name, query)">
-        </span>
-      </div>
-    </div>
-  </ng-template>
-  <div [ngClass]="{'hidden': !dataSource.selected || typing,
-                   'disable': disable}" class="wrapper-text"
-    (click)="setTyping(true)">
-    <span>
-      <label>
-        {{dataSource.selected?.name}}
-      </label>
-    </span>
-    <div class="input-group form-control">
-      <div>{{dataSource.selected?.accessId}}</div>
-    </div>
-  </div>
-  <div [ngClass]="{'hidden': dataSource.selected && !typing}">
-    <span class="field-label-wrapper">
-      <label>
-        {{dataSource.selected?.name}}
-      </label>
-    </span>
-    <div class="input-group">
-      <input #inputTypeAhead class="form-control input-text" [ngClass]="{'invalid': !dataSource.length && isRequired}"  (blur)="typeaheadOnSelect({'item':dataSource.selected})"
-        name="accessItem-{{index}}" [required]="isRequired ? 'required' : null" #accessItemName="ngModel" [(ngModel)]="value"
-        [typeahead]="dataSource" typeaheadOptionField="name" [typeaheadItemTemplate]="customItemTemplate"
-        (typeaheadOnSelect)="typeaheadOnSelect($event)" [typeaheadScrollable]="true"
-        [typeaheadOptionsInScrollableView]="typeaheadOptionsInScrollableView" [typeaheadMinLength]="typeaheadMinLength"
-        [typeaheadWaitMs]="typeaheadWaitMs" (typeaheadLoading)="changeTypeaheadLoading($event)" placeholder="{{displayError? placeHolderMessage: ''}}"
-        [@validation]="validationValue" data-cy="typeahead_input">
-      <button *ngIf="!typeaheadLoading" type="button" title="search" class="btn rounded blue search-button" >
-        <span class="material-icons md-20 blue">search</span>
-      </button>
-      <div *ngIf="typeaheadLoading" class="loading">
-        <taskana-shared-spinner [isRunning]="typeaheadLoading" positionClass="type-ahead-spinner"></taskana-shared-spinner>
-      </div>
-    </div>
-  </div>
-</div>
+<div *ngIf="dataSource" class="typeahead">
+  <label class="typeahead__label">
+    {{dataSource.selected?.name}}
+  </label>
+  <form>
+    <mat-form-field class="typeahead__form" focused="false" appearance="outline">
+      <mat-label>{{placeHolderMessage}}</mat-label>
```

Comment:
```suggestion
      <mat-label>{{dataSource.selected?.name ? dataSource.selected?.name : placeHolderMessage}}</mat-label>
```

and change this mat-label would result this:
<img width="228" alt="Screenshot 2020-10-26 at 09 28 41" src="https://user-images.githubusercontent.com/6671583/97149887-a5795b00-176d-11eb-90e1-f1b80d88fdd6.png">
<img width="219" alt="Screenshot 2020-10-26 at 09 31 26" src="https://user-images.githubusercontent.com/6671583/97150161-06a12e80-176e-11eb-90bc-b886fe4e3803.png">